extra_data/components.py

```diff
@@ -1020,10 +1149,10 @@ def _label_dims(arr):
 
         if ndim_pertrain == 4:
             arr = arr.rename({
-                'dim_0': 'pulse', 'dim_1': 'slow_scan', 'dim_2': 'fast_scan'
+                'dim_0': 'cell', 'dim_1': 'slow_scan', 'dim_2': 'fast_scan'
             })
         elif ndim_pertrain == 2:
-            arr = arr.rename({'dim_0': 'pulse'})
+            arr = arr.rename({'dim_0': 'cell'})
```

Comment:
This is technically a breaking API change (the fact you had to change the tests backs this up). It makes sense, and I think we can probably get away with this because `JUNGFRAU` support is fairly new and hopefully not that widely used yet, but normally we'd want to avoid changing something like this once it has been released.

Reply:
If you would like - we can change it back. I just thought it might produce less confusion in the future if we refrain from labeling memory cell id as pulse id just for Jungfrau data.

Reply:
I think you're right - the benefits of getting the label right *hopefully* outweigh the costs of changing it, since this class is fairly new. But it is a risk, and we should be ready to apologise if it turns out we break someone's code. :slightly_smiling_face: